common/src/Microsoft.Azure.IIoT.Serializers.NewtonSoft/src/Microsoft.Azure.IIoT.Serializers.NewtonSoft.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="Newtonsoft.Json.Schema" Version="3.0.14" />
```

Comment:
this library has an incompatible license (commercial license or AGPL).  Is it possible to use a different library here as we would not be able to get this through our governance.

Reply:
We will absolutely investigate and update. Thank you for catching that!

Reply:
@marcschier - we have updated the PR with a replacement for the library with an incompatible license. I have also updated the initial PR description to include notes about the changes we made in the subsequent commit.